src/lib/AttributeResolver.php

```diff
@@ -203,6 +203,8 @@ protected function resolveProperty(PropertySchema $property, bool $isRequired):v
                   ->setDescription($property->getAttr('description', ''))
                   ->setReadOnly($property->isReadonly())
                   ->setDefault($property->guessDefault())
+                  ->setXDbType($property->getAttr('x-db-type'))
+                  ->setNullable($property->getProperty()->getSerializableData()->nullable ?? null)
```

Comment:
why do you call `getSerializableData()` here? nullable should be accessable directly on the object. `getSerializableData()` is a significant performance overhead.

```suggestion
                  ->setNullable($property->getProperty()->nullable ?? null)
```

Reply:
That line is added by @siggi-k . Putting your suggested changes breaks few tests. I am checking this problem

Reply:
I think that solution by @siggi-k by using `getSerializableData()` is only correct way if we want to set nullable as described in [README docs](https://github.com/SOHELAHMED7/yii2-openapi/tree/100-timestamp-migrations-do-not-work-in-mariadb#handling-of-not-null-constraints) because [default value handling of nullable by OpenAPI spec](https://github.com/cebe/php-openapi/blob/master/src/spec/Schema.php#L127-L130) is somewhat different than what we want in the yii2-openapi lib

Reply:
I see, thanks for checking that. This is making the generator significantly slower on layer5, I'd like to check other ways to get the original value of nullable.

Reply:
Created another issue to handle this separately https://github.com/cebe/yii2-openapi/issues/123